### ISSUE TITLE
Code inside Plugins directory is able to import packages on local runner but MWAA plugins code couldn't

### ISSUE THREAD

**Issue by SanketSanath** [aws/aws-mwaa-local-runner#201]
I have added `confluent-kafka`, `pymongo`, `pyodbc` and other packages in requirements.txt and and importing those in code inside the plugins directory. On local runner it is working fine.

But when I zip the `plugins/` folder and move it to MWAA (also move requirements.txt) then plugins folder code couldn't import it and throws banner error
```
No module named 'pyodbc'
No module named 'pymongo'
No module named 'confluent_kafka'
```

I tried to import those inside one of the DAGs and in that condition I worked great.
```
import pyodbc
import pymongo
from confluent_kafka import Producer
```

Why is that `plugins/` folder code is not able to import third party packages on MWAA and how to solve this?

**Comment by srikanthgr** [comment]
@SanketSanath I'm facing the same issue. Did you able to resolve this?

**Comment by SanketSanath** [comment]
Nope. I have created a `utility` folder inside `dags/` folder and added all the utility function there.

**Comment by arif-banai** [comment]
Facing the same exact issue. Everything works in local-runner, but finding this error across all the files I included in my custom plugins.zip

`Broken plugin: [/usr/local/airflow/plugins/operators/custom_operator.py] No module named 'xxxxxx'`

Note the modules that are 'missing' are listed in the requirements.txt, and the requirements_install Cloudwatch log shows that the modules were installed.

**Comment by hbashary** [comment]
Anyone find a solution to this ?
